doozer/doozerlib/backend/konflux_image_builder.py

```
@@ -179,81 +178,80 @@ async def build(self, metadata: ImageMetadata):
+                image_tag = image_pullspec.split(':')[-1]
+                record["image_tag"] = image_tag
 
-                if self._config.dry_run:
-                    logger.info("Dry run: Would have inserted build record in Konflux DB")
-                else:
-                    await self.update_konflux_db(
-                        metadata, build_repo, pipelinerun_info, outcome, building_arches, build_priority
+                # Validate SLSA attestation and source image signature
+                try:
+                    await self._validate_build_attestation_and_signature(image_pullspec, metadata.distgit_key)
+                except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)